CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Changed
 
+- The Metrics SDK export record no longer contains a Resource pointer, the SDK `Export()` function for push-based exporters now takes a single Resource argument, pull-based exporters use `controller.Resource()`. (#2120)
```

Comment:
nit

in most places when some function/type is references then following "syntax" is used `"package".func/type"` e.g. `"go.opentelemetry.io/otel".SetErrorHandler`

personally, I like when the package import path is specified as "Metric SDK" might be a little cryptic for users (they may need to think a while what it is exactly)

Reply:
Fixed!